ddtrace/contrib/aioredis/__init__.py

```diff
@@ -27,6 +27,16 @@
 
    Default: ``"redis"``
 
+.. py:data:: ddtrace.config.aioredis["cmd_max_length"]
+
+   Max allowable size for the aioredis command span tag.
+   Anything beyond the max length will be replaced with ``"..."``.
+
+   This option can also be set with the ``DD_AIOREDIS_CMD_MAX_LENGTH`` environment
```

Comment:
I was really torn here on this.

We have to define it in each integration because we cannot use `config.redis.cmd_max_length` in each of these (if ddtrace.contrib.redis.patch doesn't get loaded we won't get the config value set), and we cannot localize it in `stringify_cache_args` because it is also used by Django.

We could still have each integration use `DD_REDIS_` but then is it weird to configure `DD_AIOREDIS_SERVICE` + `DD_REDIS_CMD_MAX_LENGTH`